examples/CRISP/apps/server/src/server/blockchain/handlers.rs

```diff
@@ -9,8 +9,7 @@ use crate::server::{
 };
 use chrono::Utc;
 use compute_provider::FHEInputs;
-use enclave_sdk::evm::contracts::{EnclaveContract, EnclaveRead, EnclaveWrite};
-use enclave_sdk::indexer::DataStore;
+use enclave_sdk::evm::contracts::EnclaveContract;
```

Comment:
This is causing the test to fail @hmzakhalid - looks like a merge conflict or something.
```suggestion
use enclave_sdk::evm::contracts::{EnclaveContract, EnclaveRead, EnclaveWrite};
use enclave_sdk::indexer::DataStore;
```